analyzer/parser.py

```diff
@@ -66,10 +67,10 @@ def init_zipkin_data_by_parent_span_id( parent_span_id, root, event, start_time
             zipkin_data[tmp_key] = event['val']
         return
     else:
-        for span_id in root.keys():
-            if not isinstance(span_id, int):
+        for pid in root.keys():
```

Comment:
for this one, I think we should use span_id, because we may need to put traces from different pid into one tree.